.github/workflows/asset-exchange-corda.yml

```diff
@@ -63,15 +63,15 @@ jobs:
       # CORDA NETWORK
       - name: Start Corda Network
         run: |
-          make start
+          make start-network1 PROFILE=3-nodes
           sleep 100
           docker logs corda_partya_1
           docker logs corda_partyb_1
         working-directory: tests/network-setups/corda
         
       # FABRIC CLI
       - name: Setup Corda CLI init
-        run: ./scripts/initAsset.sh
+        run: ./scripts/initAsset.sh 1
```

Comment:
What does the `1` denote here? Can the parameter value be more descriptive?

Reply:
I added that only for the purpose of this workflow.
It actually means, only to initialize for 1st network, because in workflow I'm running only one network, to reduce the load, and run tests faster.
If we don't pass any param, it will work as it used to work before.

Reply:
Can you just change the value above to something like `single-network`? That will be more comprehensible. You won't need any other code changes.